models/canadarm/urdf/SSRMS_Canadarm2.urdf.xacro

```diff
@@ -0,0 +1,352 @@
+    <origin rpy="$(arg rpy)" xyz="$(arg xyz)"/>
+  </joint>
+
+  <!-- SSRMS_Canadarm2 Dummy Base Body-->
+  <link name="Base_SSRMS">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0 0 0.5"/>
+      <mass value="243.66"/>
+      <inertia ixx="44.413" ixy="0" ixz="0" iyy="44.413" iyz="0" izz="9.336"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/ee.dae" scale="1.0 1.0 1.0" />
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/ee.stl" scale="1.0 1.0 1.0" />
+      </geometry>
+    </collision>
+  </link>
+
+  <!-- Dummy Joint Joining Base to Body B1 -->
+  <joint name="Base_Joint" type="revolute">
+    <parent link="Base_SSRMS"/>
+    <child link="B1"/>
+    <!-- Rotate Frame to Match the frames on Fig. 13 in Paper -->
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <axis xyz="0 0 1"/>
+    <limit lower="-4.71239" upper="4.71239" effort="2332" velocity="0.0698132"/>
+  </joint>
+
+
+  <!--Manipulator-->
+  <!-- Link 1 -->
+  <link name="B1">
+    <inertial>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <mass value="105.98"/>
+      <inertia ixx="12.19" ixy="0" ixz="0" iyy="12.19" iyz="0" izz="3.061"/>
+    </inertial>
+
+    <visual>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/joint_v3_0.dae" scale="1.0 1.0 1.0" />
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <geometry>
+        <!--<mesh filename="$(find simulation)/models/canadarm/meshes/joint_combine_link3.stl" scale="1.0 1.0 1.0" />-->
+        <mesh filename="$(find simulation)/models/canadarm/meshes/joint_v3_0.stl" scale="1.0 1.0 1.0" />        
+      </geometry>
+    </collision>
+  </link>
+
+  <!-- Joint 1: Shoulder Roll Joint -->
+  <joint name="Shoulder_Roll" type="revolute">
+    <parent link="B1"/>
+    <child link="B2"/>
+    <origin rpy="0 0 0" xyz="0.25082 0 -0.175"/>
+    <axis xyz="1 0 0"/>
+    <limit lower="-4.71239" upper="4.71239" effort="2332" velocity="0.0698132"/>
+  </joint>
+
+  <!-- Link 2 -->
+  <link name="B2">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0.175 0 -0.25082"/>
+      <mass value="105.98"/>
+      <inertia ixx="12.19" ixy="0" ixz="0" iyy="12.13" iyz="0" izz="3.061"/>
+    </inertial>
+
+    <visual>
+      <origin rpy="0 0 0" xyz="0.175 0 -0.25082"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/joint_v3_0.dae" scale="1.0 1.0 1.0" />
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0.175 0 -0.25082"/>
+      <geometry>
+        <!--<mesh filename="$(find simulation)/models/canadarm/meshes/joint_combine_link3.stl" scale="1.0 1.0 1.0" />-->
```

Comment:
```suggestion
```

Reply:
Deleted the commented out line, per reviewer's suggestion